packages/jazz-tools/src/tests/coMap.test.ts

```diff
@@ -543,6 +608,135 @@ describe("CoMap resolution", async () => {
     expect(loadedPerson.dog?.name).toEqual("Rex");
   });
 
+  test("loading a remotely available map with skipRetry set to true", async () => {
+    const Dog = co.map({
+      name: z.string(),
+      breed: z.string(),
+    });
+
+    const Person = co.map({
+      name: z.string(),
+      age: z.number(),
+      dog: Dog,
+    });
+
+    const currentAccount = Account.getMe();
+
+    // Disconnect the current account
+    currentAccount._raw.core.node.syncManager.getPeers().forEach((peer) => {
+      peer.gracefulShutdown();
+    });
+
+    const group = Group.create();
+    group.addMember("everyone", "writer");
+
+    const person = Person.create(
+      {
+        name: "John",
+        age: 20,
+        dog: Dog.create({ name: "Rex", breed: "Labrador" }, group),
+      },
+      group,
+    );
+
+    const userB = await createJazzTestAccount();
+    let resolved = false;
+    const promise = Person.load(person.id, {
+      loadAs: userB,
+      skipRetry: true,
+    });
+    promise.then(() => {
+      resolved = true;
+    });
+
+    expect(resolved).toBe(false);
+
+    await new Promise((resolve) =>
+      setTimeout(
+        resolve,
+        cojsonInternals.CO_VALUE_LOADING_CONFIG.RETRY_DELAY - 100,
+      ),
+    );
+
+    expect(resolved).toBe(true);
```

Comment:
This kind of time-based tests are very likely to be flaky on CI when the machines are slow.

A possible way to test this is to set the `cojsonInternals.CO_VALUE_LOADING_CONFIG.RETRY_DELAY` to a very high value and check the the load resolves to `unavailable` (should be null here)